plugins/europeana/record.js

```diff
@@ -70,4 +72,23 @@ function getRecord(europeanaId, params) {
     });
 }
 
+/**
+ * Find the content type of a file and if type is a pdf return the url
+ * @param {string} url of a file
+ * @return {Object} the url of the file
+ */
+function findPDFContent(file, webResources) {
+  let mimeType;
+  for (let resource in webResources) {
+    if (file === webResources[resource].rdfAbout) {
+      mimeType = webResources[resource].ebucoreHasMimeType;
+    }
+  }
+
+  if (mimeType && mimeType.includes('pdf')) {
```

Comment:
Let's do a strict match on `=== 'application/pdf'` as that is the only formal MIME type for PDFs.